src/analysis/visualize.py

```diff
@@ -0,0 +1,45 @@
+import numpy as np
+np.random.seed(7)
+import matplotlib.pyplot as plt
+
+import seaborn as sns
+sns.set(style="white", palette="muted", color_codes=True, context="talk")
+
+create_gif = False
+
+"""
+    Show the results in the grahph plot.
+"""
+
+def plot_distributions(y, Z, iteration=None, val_metrics=None, p_rules=None, fname=None):
+    fig, axes = plt.subplots(1, 1, figsize=(10, 4), sharey=True)
+    legend = {'race': ['African-American', 'Others']}
+    for idx, attr in enumerate(Z.columns):
+        for attr_val in [0, 1]:
+            ax = sns.distplot(y[Z[attr] == attr_val], hist=False,
+                              kde_kws={'shade': True, },
+                              label='{}'.format(legend[attr][attr_val]))
+        ax.set_xlim(0, 1)
+        ax.set_ylim(0, 7)
```

Comment:
Why is the `7` hard-coded in? What does it represent?

Reply:
In general, preferable to have a function like `ax.set_ylim(0, max(y_data))` or something like that.

Reply:
Will surely look into these hard-coded value and fix them in the next PR